trlx/trainer/__init__.py

```diff
@@ -11,17 +11,17 @@
 from trlx.utils import safe_mkdir
 
 # specifies a dictionary of architectures
-_MODELS: Dict[str, Any] = {}  # registry
+_TRAINERS: Dict[str, Any] = {}  # registry
 
 
-def register_model(name):
-    """Decorator used register an architecture
+def register_trainer(name):
+    """Decorator used to register a trainer
```

Comment:
unrelated but a lot of these comment strings havent been changed since they were copied over from magiCARP ;)

Reply:
🎣